api/v1alpha1/clientprofilemapping_types.go

```diff
@@ -20,29 +20,22 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// BlockPoolRefSpec identify a blockpool - client profile pair
-type BlockPoolRefSpec struct {
-	//+kubebuilder:validation:Required
-	ClientProfileName string `json:"clientProfileName,omitempty"`
-
-	//+kubebuilder:validation:Required
-	//+kubebuilder:validation:Minimum:=0
-	PoolId int `json:"poolId,omitempty"`
-}
-
-// BlockPoolMappingSpec define a mapiing between a local and remote block pools
-type BlockPoolMappingSpec struct {
-	//+kubebuilder:validation:Required
-	Local BlockPoolRefSpec `json:"local,omitempty"`
-
-	//+kubebuilder:validation:Required
-	Remote BlockPoolRefSpec `json:"remote,omitempty"`
+// MappinsSpec define a mapiing between a local and remote profiles 
```

Comment:
MappingsSpec define a mapipng between a local and remote profiles

Reply:
fixed